### PR TITLE
Feature - TRSV2-190 - Feature Flags

### PR DESCRIPTION
**Pull request by chris-pettinga**

### CODE REVIEW

File: trade_remedies_api/config/feature_flags.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from flags import conditions
+
+
+def part_of_group_condition(request, **kwargs):
+    return kwargs["user_object"].is_authenticated and kwargs["user_object"].groups.filter(
```
Comment (holdenweb) [inline trade_remedies_api/config/feature_flags.py]: docstring

**Review by holdenweb — APPROVED** [review]
Throughly reviewed. The usual naming questions (and absence of docstrings) but functionally sound.

**Comment by holdenweb** [comment]
Looks like a file was left unblackened. Sorry!

File: trade_remedies_api/config/renderers.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+
+class APIResponseRenderer(JSONRenderer):
+    def render(self, data, accepted_media_type=None, renderer_context=None):
+        response_dict = {"success": True, "response": {}}
+        if hasattr(data, "serializer"):
+            # We're just returning one serializer, we need to find out if it contains multiple
+            # objects or just one. We can do this by checking the many argument
+            response_type = (
+                "results" if hasattr(data.serializer, "many") and data.serializer.many else "result"
```
[the remark this answers is not in the document]
Comment (chris-pettinga) [reply trade_remedies_api/config/renderers.py:12]: Yes I do agree with you in general, however the current API client does throw errors depending on if it receives "result" or "results" versus what it's meant to.  Essentially it's made to work only with the current Response JSON structure.

With the completion of https://uktrade.atlassian.net/jira/software/projects/TRSV2/boards/288?selectedIssue=TRSV2-238, we will have a new API client which will not be so fickle, meaning we can scrap this custom JsonRenderer

**Pull request merged**